README.md

```diff
@@ -437,12 +439,12 @@ As all of these done, start the local server by `npm start` to launch the dapp.
 
 # Run in cita-web-debugger
 
-[cita-web-debugger](https://github.com/cryptape/cita.js/tree/develop/packages/cita-web-debugger) is an CITA Web Debugger on Chrome, acts as an CITA Wallet to sign transactions from DApp.
+[cita-web-debugger](https://github.com/cryptape/cita.js/tree/develop/packages/cita-web-debugger) is is an extension of chrome and a debugger tool for CITA.
```

Comment:
double `is`

Reply:
if the object before and after `and` refers to the same one, the second `a` should be omitted